guide/selenium/src/test/scala/io/udash/web/guide/demos/ext/I18nDemosTest.scala

```diff
@@ -1,37 +1,36 @@
 package io.udash.web.guide.demos.ext
 
 
+import com.avsystem.commons.jiop.JavaInterop._
 import io.udash.web.SeleniumTest
-import org.openqa.selenium.By.{ById, ByTagName}
-import org.openqa.selenium.WebElement
-import org.scalactic.source.Position
+import org.openqa.selenium.By.{ByCssSelector, ById}
 
 class I18nDemosTest extends SeleniumTest {
   val url = "/ext/i18n"
 
-  private def runDemo(id: String)(additionalAction: WebElement => Unit = _ => ())(implicit position: Position): Map[String, String] = {
-    def demo = {
-      driver.switchTo().parentFrame()
-      driver.findElementById(id)
-    }
-
-    eventually {
-      driver.switchTo().frame(demo.findElement(new ByTagName("iframe")))
-      driver.findElement(new ById("run-icon")).click()
-    }
-
-    eventually {
-      driver.switchTo().frame(demo.findElement(new ByTagName("iframe")))
-      driver.switchTo().frame(driver.findElement(new ById("codeframe")))
-      val output = driver.findElementById("output")
-      additionalAction(output)
-      val translations = output.getText.split("\n").collect {
-        case line if line.indexOf(":") != -1 => (line.take(line.indexOf(":")), line.drop(line.indexOf(":") + 2))
-      }.toMap
-      translations.size should be(6)
-      translations
-    }
-  }
+  //private def runDemo(id: String)(additionalAction: WebElement => Unit = _ => ())(implicit position: Position): Map[String, String] = {
```

Comment:
do we really need it? How about some comment with todo then?

Reply:
I wanted to save time for when new rpc is merged. Adding comment.